package.json

```diff
@@ -84,7 +84,7 @@
     "watch-css": "npm run build-css && node-sass-chokidar src/ -o src/ --watch --recursive",
     "start-js": "react-scripts start",
     "check-version": "node ./scripts/check-version.js",
-    "start": "npm-run-all -p watch-css start-js check-version",
+    "start": "NODE_OPTIONS=--max-old-space-size=5120 npm-run-all -p watch-css start-js check-version",
```

Comment:
This syntax doesn't work in Windows. One possible fix is to add the `cross-env` library and make the call `cross-env NODE_OPTIONS=--max-old-space-size=5120 npm-run-all -p watch-css start-js check-version`.  

That said, it looks like this is only necessary when using the larger mock data set (available separately per licensing), not the one that's in the repo.

Reply:
@rdingwell Thoughts?

Reply:
@dehall does the addition cause it not to run on windows?

Reply:
@noranda the env setting crashes on windows.   Seeing how it's only needed large static data sets why dont we just remove it for now and set that env locally

Reply:
Yes it prevents `yarn start` from running: 
> λ yarn start
yarn run v1.3.2
$ NODE_OPTIONS=--max-old-space-size=5120 npm-run-all -p watch-css start-js check-version
'NODE_OPTIONS' is not recognized as an internal or external command,
operable program or batch file.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Reply:
I'll go ahead an remove this and push the update

Reply:
I removed and pushed.